src/Inventory/Conf.php

```diff
@@ -216,8 +217,15 @@ protected function importContentFile(Request $inventory_request, $path, $content
             $inventory_request->handleContentType($mime);
             $inventory_request->handleRequest($contents);
             if ($inventory_request->inError()) {
+                $response = $inventory_request->getResponse();
+                if ($inventory_request->getMode() === Request::JSON_MODE) {
+                    $response = json_decode($inventory_request->getResponse());
+                } else {
+                    $xml = simplexml_load_string($response);
+                    $response = $xml->ERROR;
+                }
                 Session::addMessageAfterRedirect(
-                    __('File has not been imported:') . " " . $inventory_request->getResponse(),
+                    __('File has not been imported:') . " " . Sanitizer::encodeHtmlSpecialChars($response),
```

Comment:
If `$response` has been truncated by `Html::resume_text()`, it will contains `&nbsp;`. `Sanitizer::encodeHtmlSpecialChars()` will re-encode this non-breakale space, so user will see `&nbsp;` in the UI.